### PR TITLE
Proposal: Deprecate some tests

### PR DESCRIPTION
**Pull request by rejas**
This PR marks these tests as deprecated and to be removed with v3.9;

Unicode (https://github.com/Modernizr/Modernizr/issues/2468) has dubious value 
Touchevents (https://github.com/Modernizr/Modernizr/pull/2432) is often misinterpreted

### CODE REVIEW

**Comment by roblarsen** [comment]
This might be a better solution that outright removing the tests. `console.warn`?

**Comment by rejas** [comment]
Doy ou mean a warning when building or running the tests?
As for removing: Mabye a major release (4.0) would be better, since those can mean breaking changes per semver definition?

**Comment by roblarsen** [comment]
Building would be less intrusive and would target the developer using the tests. 

And yeah, if they're removed a major release would be the way to go.

File: README.md [hunk 1]
```diff
@@ -104,6 +104,14 @@ http://localhost:8080/test/unit.html
 http://localhost:8080/test/integration.html
 ```
  
+## Deprecation
+
+These tests are considered deprecated. They are not included anymore in the default build
+and will re removed in the next major version:
```
Comment (paulirish) [inline README.md]:
```suggestion
and will be removed in the next major version:
```

**Comment by rejas** [comment]
After looking at the code, I am not the wiser where to add a warning when building modernizr...

Anyway, merging this for 3.8 would be a thing and adding a better warning one for 3.9 :-) Any objections?

**Pull request merged**